processor/src/host/mod.rs

```diff
@@ -214,63 +210,64 @@ impl<A: AdviceProvider> Host for DefaultHost<A> {
     }
 
     fn on_event(&mut self, process: ProcessState, event_id: u32) -> Result<(), ExecutionError> {
-        if event_id == SystemEvent::FalconSigToStack.into_event_id() {
-            // provide a default implementation for handling FalconSigToStack event since it is not
-            // handled any more by the system event handlers. the handler assumes that the private
-            // key is in the advice provider and uses it to in signature generation
+        #[cfg(any(test, feature = "testing"))]
+        if event_id == crate::utils::EVENT_FALCON_SIG_TO_STACK {
             let advice_provider = self.advice_provider_mut();
-            push_signature(advice_provider, process, SignatureKind::RpoFalcon512)
-        } else {
-            #[cfg(feature = "std")]
-            std::println!(
-                "Event with id {} emitted at step {} in context {}",
-                event_id,
-                process.clk(),
-                process.ctx()
-            );
-            Ok(())
+            return test::push_falcon_signature(advice_provider, process);
         }
```

Comment:
I put this here since our `Test` struct uses `DefaultHost` instead of some `TestHost`. But I don't think anyone uses `DefaultHost` anyways, so its main use is probably just tests

Reply:
Done